sellItem.js

```diff
@@ -1,8 +1,29 @@
 const defectsChoicesInSwedish = new Map().set("hole", "Hål").set("stain", "Fläck").set("lostFit", "Tappad passform").set("nopprig", "Nopprig").set("threadUp", "Trådsläpp").set("colorChange", "Färgändring").set("otherDefect", "Annat");
 const imageElements = ["frontImage", "brandTagImage", "defectImage", "materialTagImage", "extraImage"];
 
+async function requestUniqueId() {
+  const endpointUrl = 'https://generateuniqueid-heypmjzjfq-ew.a.run.app';
+  try {
+    const response = await fetch(endpointUrl, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+    if (!response.ok) {
+      console.error(`Error: ${response.statusText}`);
+      return null;
+    }
+    const data = await response.json();
+    return data.id;
+  } catch (error) {
+    console.error(`Failed to fetch unique ID: ${error.message}`);
+    return null;
```

Comment:
Perhaps falling back to `return uuidv4()` here in case of temporary network problems?